composer.json

```diff
@@ -1,29 +1,29 @@
 {
-    "name": "joomla-projects/joomla-browser",
-    "description": "joomla-browser Codeception Module",
-    "keywords":["BDD", "acceptance testing", "tdd", "joomla"],
-    "homepage":"https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception",
-    "license": "GPL-2.0+",
-    "authors": [
+    "name"          : "joomla-projects/joomla-browser",
+    "description"   : "joomla-browser Codeception Module",
+    "keywords"      : ["BDD", "acceptance testing", "tdd", "joomla"],
+    "homepage"      : "https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception",
+    "license"       : "GPL-2.0+",
+    "authors"       : [
         {
-            "name": "Puneet Kala",
-          "email": "puneet.kala@community.joomla.org"
+          "name"    : "Puneet Kala",
+          "email"   : "puneet.kala@community.joomla.org"
         },
         {
-          "name": "Javier Gomez",
-          "email": "javier.gomez@community.joomla.org"
+          "name"    : "Javier Gomez",
+          "email"   : "javier.gomez@community.joomla.org"
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php"                                   : ">=5.5.9",
```

Comment:
On a second thought, no workaround will be needed after all, since all Selenium clients (with Robo) will run in latest PHP 7.x, while only servers will vary Joomla version, so it should not be an impediment for anything.